spread/build/sbuild/task.yaml

```diff
@@ -49,6 +49,7 @@ execute: |
       --dist=${RELEASE}
       --build=${DEB_BUILD_ARCH}
       --host=${DEB_HOST_ARCH}
+      --purge=never
```

Comment:
In my recent experience this still removes all the packages from the chroot, doesn't it? Do you know a way to avoid that as well?

Reply:
Hm. I thought that would suppress that as well.

`--purge-build-deps` is probably the option needed, but (a) I thought that `--purge` implied that, and at least *some* of the `--purge-*` options are mutually exclusive 🤷‍♀️

Reply:
Maybe I used `--purge-session=never`…